src/components/project.js

```diff
@@ -16,21 +18,22 @@ const Project = ({
   additionalDescription,
   callToAction,
 }) => (
-  <Card>
-    <Img fluid={image} alt="" />
-    <Card.Title>{title}</Card.Title>
-    <Card.Body>
-      <Card.Text>{primaryPartner}</Card.Text>
-      <Card.Text>{otherPartners}</Card.Text>
-      <Card.Text>{description}</Card.Text>
-      <Card.Text>
-        <ReactMarkdown source={additionalDescription} />
-      </Card.Text>
-      <Card.Text>
-        <Linkify>{callToAction}</Linkify>
-      </Card.Text>
-    </Card.Body>
-  </Card>
-)
-
+    <Container className="partners">
+      <Card className="partners-card">
+        <Card.Img variant="top" src={image} className="partners-image" />
```

Comment:
The problem is here. You need to use the Img component from gatsby, not react-bootraps Card.Img. 

I know, its confusing but its tru

Reply:
So here, just change `src={image}` to `src={Img}` ?